assets/js/components/notifications/CoreSiteBannerNotifications.js

```diff
@@ -75,30 +75,44 @@ const CoreSiteBannerNotifications = () => {
 		return null;
 	}
 
-	return notifications.map( ( notification ) => (
-		<BannerNotification
-			key={ notification.id }
-			id={ notification.id }
-			title={ notification.title || '' }
-			description={ notification.content || '' }
-			learnMoreURL={ notification.learnMoreURL || '' }
-			learnMoreLabel={ notification.learnMoreLabel || '' }
-			ctaLink={ notification.ctaURL || '' }
-			ctaLabel={ notification.ctaLabel || '' }
-			ctaTarget={ notification.ctaTarget || '' }
-			dismiss={
-				notification.dismissLabel ||
-				__( 'OK, Got it!', 'google-site-kit' )
-			}
-			isDismissible={ notification.dismissible }
-			onCTAClick={ async () => {
-				await acceptNotification( notification.id );
-			} }
-			onDismiss={ async () => {
-				await dismissNotification( notification.id );
-			} }
-		/>
-	) );
+	const MappedNotification = ( { notification } ) => {
```

Comment:
Let's extract it into a separate component `CoreSiteBannerNotification`.

Reply:
https://github.com/google/site-kit-wp/pull/4349/commits/2c72deda97e21244b424a4ffa8b39bab901f6ec8